src/Uuid/FieldUuid.php

```diff
@@ -108,13 +110,16 @@ public static function index(): Generator
 
 	/**
 	 * Returns value to be stored in cache,
-	 * constisting of three parts:
+	 * consisting of three parts:
 	 * - parent UUID including scheme
 	 * - field name
 	 * - UUID id string for model
 	 */
 	public function value(): array
 	{
+		/**
+		 * @var \Kirby\Cms\Site|\Kirby\Cms\Page|\Kirby\Cms\User $model
```

Comment:
`Question`: Couldn't it also be a file?

Reply:
You are right. It seems I was rather thinking about the result of `$model->parent()` here.